README.md

```diff
@@ -12,15 +12,22 @@ Compile OOMMF on (Debian) Linux with spack:
 
 [![debian-spack-v0.16.2](https://github.com/fangohr/oommf-in-spack/actions/workflows/debian-spack-v0.16.2.yml/badge.svg)](https://github.com/fangohr/oommf-in-spack/actions/workflows/debian-spack-v0.16.2.yml)
 
-For information: (not part of this `main` branch)
+[![debian-spack-v0.16.3](https://github.com/fangohr/oommf-in-spack/actions/workflows/debian-spack-v0.16.3.yml/badge.svg)](https://github.com/fangohr/oommf-in-spack/actions/workflows/debian-spack-v0.16.3.yml)
+
+
+
+Compile OOMMF on OSX with Spack's `develop` branch (see branch https://github.com/fangohr/oommf-in-spack/tree/osx for these workflows):
```

Comment:
Could we maybe say that there are some additional branches for testing "more specific use cases" (`develop`, `osx`) that are (for simplicity or information purposes ?) listed below?